docs/operating-scylla/admin-tools/index.rst

```diff
@@ -21,6 +21,7 @@ Admin Tools
    Reading mutation fragments </operating-scylla/admin-tools/select-from-mutation-fragments/>
    Maintenance socket </operating-scylla/admin-tools/maintenance-socket/>
    Maintenance mode </operating-scylla/admin-tools/maintenance-mode/>
+   Task manager </operating-scylla/admin-tools/task-manager/>
```

Comment:
I don't see this in the preview for some reason. Maybe need to add this in another place too?

Reply:
I still don't see it.

BTW, we already have a REST API page: operating-scylla/rest/
I don't think we should add this new docs there, because no one will find it. But the task-manager page should link to this page, for instructions on how to interact with the API.

Reply:
You need to reload the page

Reply:
Yes, it was probably my browser cache. It works now.